pyoos/collectors/nerrs/nerrs_soap.py

```diff
@@ -30,6 +45,8 @@ def get_stations(self):
                 if val is None:
                     val = ""
                 s[child.tag] = val
+            s['Longitude'] = -float(s['Longitude'])
```

Comment:
Just made the change.  We need to find a way to test this PR against a working NERRS configuration.

> Just remember: pyoos is not necessarily ingesting data sources that follow community standards. It takes important data sources in their existing forms, and typically ingests and transforms them into its implementation of Unidata CDM DSG (well, except for NDBC and probably COOPS ...). Idiosyncracies are to be expected at the source.

One more reason to enforce negative longitudes instead of hoping they are always positive.